app/models/external_post.rb

```diff
@@ -1,11 +1,6 @@
 class ExternalPost < Post
-	self.table_name = 'external_posts'
-
 	validates_uniqueness_of :external_id
-
-	private
-
-	def child?
-	  true
+	def username
+		self.user_display_name
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Redundant `self` detected.<br>Tab detected.